packages/extension-browser/src/devtools/views/pages/results/problem.tsx

```diff
@@ -23,8 +23,11 @@ const Problem = ({ problem, index }: Props) => {
 
     const onInspectElementClick = useCallback(() => {
         // Verify elementId is actually a number since it originates from untrusted snapshot data.
+
         if (typeof elementId === 'number') {
-            evaluate(`inspect(__webhint.findNode(${elementId}))`);
+            evaluate(`inspect(__webhint.findNode(${elementId}))`, () => {
+                evaluate(`$0.scrollIntoView()`);
```

Comment:
Think we could put this in a single `evaluate` call? Here's what I'm thinking (I also added a check for the API since `node` is allowed to be a DOM node other than an `Element`, but only `Element` will have `scrollIntoView`):
```ts
evaluate(`(function() {
    var node = __webhint.findNode(${elementId});
    inspect(node);
    if (node && node.scrollIntoView) {
        node.scrollIntoView();
    }
})()`);
```